src/shop-app.html

```diff
@@ -18,11 +18,25 @@
 <link rel="import" href="../bower_components/iron-media-query/iron-media-query.html">
 <link rel="import" href="../bower_components/iron-pages/iron-pages.html">
 <link rel="import" href="../bower_components/iron-selector/iron-selector.html">
+<link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
 <link rel="import" href="shop-category-data.html">
 <!-- Since 'home' is the default route, eagerly load it. -->
 <link rel="import" href="shop-home.html">
 
+
+
 <dom-module id="shop-app">
+  <!--
+    TODO(ricitc): make these lazy-imports after
+        https://github.com/Polymer/polymer-analyzer/pull/543 lands
+        DO NOT MERGE until then :)
+  -->
+  <link rel="import" href="shop-cart-data.html">
+  <link rel="import" href="lazy-resources.html">
+  <link rel="import" href="shop-list.html">
+  <link rel="import" href="shop-detail.html">
+  <link rel="import" href="shop-cart.html">
+  <link rel="import" href="shop-checkout.html">
```

Comment:
> By fragments do you mean the DOM to display for each possible choice in iron-pages?

No I meant the fragments as specified in the `polymer.json`

Yes I agree that they are separate concerns, which should be specified, but if these are in different places people forget things. We have hit this exact problem multiple times, in which I built custom scripts to fix that. Yes these custom scripts do use `iron-lazy-pages`, since we are using them, but at least they give more security for us as application developers that stuff doesn't break when we deploy.

I feel that these custom scripts should be integrated somewhere, without too much hassle for the user. Moreover, the same mechanic can be used to analyze the lazy-fragments.